docs/imagecustomizer/api/configuration/injectArtifactMetadata.md

```diff
@@ -58,16 +58,30 @@ Added in v0.14.
 
 Required.
 
-Path to the signed artifact file to be injected. This path may be relative to the
+Path to the artifact file to be injected. This path may be relative to the
 `inject-files.yaml` config file or an absolute path.
 
+The file at this path should be initially the unsigned artifact extracted during
```

Comment:
This sounds like instructions for the user, instead of the behavior of the tool.

I would rephrase it to:

`When the config file is generated by the output artifacts API, this will be the path of the unsigned artifact. Users should replace this file with the signed version before running the inject-files command.`